crates/orchestrator/src/cli/aws_config.rs

```diff
@@ -0,0 +1,29 @@
+use clap::Args;
+use serde::Serialize;
+use url::Url;
+
+/// Parameters used to config AWS.
+#[derive(Debug, Clone, Args, Serialize)]
+#[group(requires_all = ["aws_access_key_id", "aws_secret_access_key", "aws_region"])]
+pub struct AWSConfigCliArgs {
```

Comment:
why not a separate folder for this? also no options? is it because AWS is mandatory for now (although others are mandatory as well for now)

Reply:
Separated to a new folder `Provider`